string prop_label
float64 theta_small
float64 theta_large